docs/guide/site_factory.md

```diff
@@ -286,6 +286,24 @@
 
     User Group skeletons are retained after deleting the site.
     
+    ### Define Roles to be automatically updated when a site is created  
+    
+    Role definitions can contain user/login policies with limitations that limit user access to certain sites. 
+    To avoid the need to revisit all Roles and add the newly created SiteAccess to a list of limitations, you can decide that the Roles you select are automatically updated when the site is created.
```

Comment:
```suggestion
    To avoid the need to revisit all Roles and add the newly created SiteAccess to a list of Limitations,
    you can decide that the Roles you select are automatically updated when the site is created.
```